back-end/lms/models/student.py

```diff
@@ -17,22 +49,47 @@ class Meta:
 
 class Program(models.Model):
     code = models.CharField(primary_key=True, max_length=8)
-    program = models.CharField(max_length=500, blank=True, null=True)
+    program = models.CharField(max_length=128, blank=True, null=True)
```

Comment:
Is this the title of the `Program`?

Reply:
Yes, this is our old model. Might be better to call this field title, true